readme.md

```diff
@@ -50,6 +50,9 @@ Throws an error if the macOS version is not greater than or equal to the specifi
 
 *Prefer this over `.assert()` whenever possible as it outputs a more user-friendly error message.*
 
+### assertMacOS()
+
+Throws an error if platform is not OSX/macOS (i.e. darwin).
```

Comment:
Should be:

> Throws an error if the platform is macOS.

Reply:
Really? That seems backward to me. I am reading this function as, for example:

```
console.assert(isMacOS)
```

which wouldn't throw on a mac, but would on other platforms.

Reply:
Sorry, forgot the `not`. Meant:

> Throws an error if the platform is not macOS.